src/components/screens/dashboard/dashboard-screen.tsx

```diff
@@ -1,18 +1,57 @@
-import { useAtom } from "jotai";
-import { authAtom } from "../../../atoms/auth";
+import { useAtomValue, useSetAtom } from "jotai";
+import { authAtom, userProfileAtom } from "../../../atoms/auth";
+import { useEffect, useState } from "react";
+import { useApi } from "../../../hooks/use-api";
+import { PersonTotalTime } from "../../../generated/client";
+import { errorAtom } from "../../../atoms/error";
+import LoaderWrapper from "../../generics/loader-wrapper";
+import { getHoursAndMinutes } from "../../../utils/time-utils";
 
 /**
  * Dashboard screen component
- * 
  */
-const DashboardScreen = () => {
-  const [auth] = useAtom(authAtom);
+function DashboardScreen () {
+  const auth = useAtomValue(authAtom);
+  const userProfile = useAtomValue(userProfileAtom);
+  const setError = useSetAtom(errorAtom);
+  const [isLoading, setIsLoading] = useState(false);
+  const [personTotalTime, setPersonTotalTime] = useState<PersonTotalTime[]>([]);
```

Comment:
Does personTotalTime need to be an array, if not this could be changed and you would not need to map in the component return (which could be a bit confusing if there is only ever one PersonTotalTime in this case.